server/methods/followUser.js

```diff
@@ -0,0 +1,43 @@
+import { Meteor } from 'meteor/meteor';
+
+import { Users } from '../../app/models';
+import { settings } from '../../app/settings';
+
+
+Meteor.methods({
+	followUser(username) {
+		if (settings.get('Newsfeed_enabled')) {
+			// Update followers keys
+			let userObject = Users.findOneByUsername(username);
+			if ('followers' in userObject) {
+				const followersObject = userObject.followers;
+				const { _id } = Users.findOneByUsername(Meteor.user().username);
```

Comment:
You can directly use `Meteor.userId()`

Reply:
`Meteor.userId()` is the best approach here, but just for your knowledge you could have used `Meteor.user()._id` since `Meteor.user()` will return the entire user's object, so on that case you are requesting the entire object twice.

And, if you want just one field from a query, please always pass that information to the query to not transfer unnecessary data from db to rocket.chat:
```js
Users.findOneByUsername(Meteor.user().username, {fields: {_id: 1}});
```